heartbeat/monitors/wrappers/summarizer/summarizer.go

```diff
@@ -91,7 +91,7 @@ func (s *Summarizer) Wrap(j jobs.Job) jobs.Job {
 	return func(event *beat.Event) ([]jobs.Job, error) {
 		conts, jobErr := j(event)
 
-		_, _ = event.PutValue("monitor.check_group", s.checkGroup)
+		_, _ = event.PutValue("monitor.check_group", fmt.Sprintf("%s-%d", s.checkGroup, s.jobSummary.Attempt))
```

Comment:
@andrewvc Sorry for the delayed note on this PR, Would this break the existing uptime UI in any way.

Reply:
No, the UI treats the UUID as opaque values, as does ES (it's a keyword type, there's no dedicated UUID type).